ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -295,59 +295,50 @@ TorchCore::getTensorTypeToTorch (tensor_type tensorType, torch::Dtype *torchType
 }
 
 /**
- * @brief	check the inserted information about output tensor with model
+ * @brief	check the inserted information about idx-th output tensor with model
  * @return 0 if OK. non-zero if error.
  *        -1 if the number of output tensors is not matched.
  *        -2 if the type of output tensors is not matched.
  *        -3 if the dimension of output tensors is not matched.
  */
 int
-TorchCore::validateOutputTensor (const at::Tensor output)
+TorchCore::validateOutputTensor (const at::Tensor output, unsigned int idx)
 {
   auto tensor_shape = output.sizes ();
+  tensor_type otype;
+  gsize num_gst_tensor, num_torch_tensor;
+  at::Tensor sliced_output = output.slice (0);
+  c10::IntArrayRef sliced_output_sizes = sliced_output.sizes ();
 
-  if (tensor_shape[0] != 0 && outputTensorMeta.num_tensors != tensor_shape[0]) {
-    ml_loge ("Invalid output meta: different size");
-    return -1;
-  }
-
+  /** when output is a scalar */
   if (tensor_shape[0] == 0) {
     tensor_type otype = getTensorTypeFromTorch (output.scalar_type ());
```

Comment:
duplicated declaration 
`otype = getTensorTypeFromTorch (output.scalar_type ());`

Reply:
Thanks! I fixed the code.